scripts/generate_users.py

```diff
@@ -0,0 +1,98 @@
+import sys
+import csv
+import decimal
+import random
+import datetime
+import argparse
+
+from faker import Faker
+from faker.providers import BaseProvider
+
+
+class UserDataProvider(BaseProvider):
+
+    def date(self):
+        start_date = datetime.date(2018, 1, 1)
+        return start_date + datetime.timedelta(random.randint(1, 365))
```

Comment:
Would it be better to use random numbers with a fixed seed to get reproducible test data? Or does faker do this automatically?

Reply:
Previously, the data was generated each time without any assertions for the test. I have now migrated the test data generation into a CLI tool to ensure reusability, if needed. However, the actual test data that is used for testing purposes will be stored in a CSV, and thus, will not change.